src/styles/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  padding: 16px;
+  padding-top: 32px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
```

Comment:
not sure if that's right, but if you add a height from scratch, it will be a perfect match for test

Reply:
yes, you can add height for image according to the design